src/meshapi/tests/test_admin_list_view.py

```diff
@@ -0,0 +1,34 @@
+from django.test import Client, TestCase
+from django.contrib.auth.models import User
+
+
+# Sanity check to make sure that the list views in the admin panel still work
+# These will often break when you update something in the model and forget to
+# update the admin panel
+class TestAdminListView(TestCase):
```

Comment:
Can we please test search views as well? Currently list views work but searches don't